components/actiongroup/index.css

```diff
@@ -75,24 +70,30 @@ governing permissions and limitations under the License.
       z-index: 0;
 
       &:first-child {
+        /* Action button passthrough styling */
+        --spectrum-actionbutton-focus-ring-border-radius: var(--mod-actiongroup-border-radius, var(--spectrum-actiongroup-border-radius)) 0px 0px var(--mod-actiongroup-border-radius, var(--spectrum-actiongroup-border-radius));
```

Comment:
This variable does not exist on action button so I'm assuming it should be `--mod-actionbutton-focus-indicator-border-radius`. I also noticed you're not using the `mod` version of the variable. Should we default to using mod or to just overwriting the existing variable?

Reply:
Ok that sounds good! I can write up a quick bug card to get that focus indicator issue tracked.